src/app/outbound/mod.rs

```diff
@@ -237,22 +239,19 @@ pub fn spawn<R, P>(
         .push(strip_header::request::layer(super::L5D_CLIENT_ID))
         .push(strip_header::request::layer(super::DST_OVERRIDE_HEADER))
         .push(insert::target::layer())
+        .push(trace::layer(|addr: &Addr| info_span!("addr", %addr)))
         .push_buffer_pending(max_in_flight, DispatchDeadline::extract)
         .push(router::layer(
-            router::Config::new("out addr", capacity, max_idle_age),
+            router::Config::new(capacity, max_idle_age),
             |req: &http::Request<_>| {
                 super::http_request_l5d_override_dst_addr(req)
                     .map(|override_addr| {
-                        debug!("outbound addr={:?}; dst-override", override_addr);
+                        debug!("using dst-override");
```

Comment:
Nit: should this record the override addr as well?
```suggestion
                        debug!(?override_addr, "using dst-override");
```